runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5175,6 +5175,11 @@ class INTERPRETER_CLASS
 		returnState = (I_32 *)(UDATA)*(I_64 *)(_sp + 7); /* returnStateMemAddr */
 #endif /* JAVA_SPEC_VERSION >= 20 */
 
+#if JAVA_SPEC_VERSION >= 21
+		/* Set the linker option to the current thread for the trivial downcall. */
+		_currentThread->isInTrivialDownCall = (0 == *(U_32*)(_sp + 9)) ? FALSE : TRUE; /* isInTrivialDownCall */
```

Comment:
The trailing comment doesn't contribute anything; please remove it.

Reply:
Removed.